echidna/scripts/dump_spectra_ntuple.py

```diff
@@ -20,94 +19,82 @@
 import argparse
 import csv
 import echidna.output.store as store
+import echidna.core.spectra as spectra
 import echidna.core.fill_spectrum as fill_spectrum
-import echidna.output.plot as plot
+import echidna.output.plot_root as plot
 
 
-def read_and_dump_ntuple(fname, half_life, spectrum_name, save_path):
+def read_and_dump_ntuple(fname, config_path, spectrum_name, save_path):
```

Comment:
Can you add `**kwargs` to this method, which then get passed to `fill_spectrum.fill_from_ntuple`, so you can pass in keyword arguments here that are used in `fill_from_ntuple` and the extractors, e.g. `bipo`, 'fv_radius' and 'outer_radius'.